inventories/seapath_basic_cluster_definition_example.yml

```diff
@@ -5,15 +5,16 @@ all:
             children:
                 hypervisors:
                     hosts:
```

Comment:
This review is concerning the commit message and the commit itself:

1. A good practice in git commit is 1 commit = 1 modification (better for reading latter what your commit does)
So I would suggest you to split this commit into 2 commits
2. On your commit title, a good practice is "<FILE NAME>: <COMMIT PURPOSE> (=what this commit does)"